### DISCUSSION TITLE
Goldman Sachs SDE role (Open to new grads)

### DISCUSSION THREAD

**Discussion by quantumscholar** [ReaVNaiL/New-Grad-2024#480] [General]
https://hdpc.fa.us2.oraclecloud.com/hcmUI/CandidateExperience/en/sites/LateralHiring/job/116718?mode=job&iis=LinkedIn&utm_medium=jobshare

**Comment by ReaVNaiL** [comment]
Goldman Sachs role is currently open in the list.
